addon-resizer/Makefile

```diff
@@ -21,13 +21,28 @@ ALL_ARCH = amd64 arm arm64 ppc64le s390x
 ARCH ?= amd64
 export DOCKER_CLI_EXPERIMENTAL=enabled
 
+BUILDDIR=$(CURDIR)
+registry_url ?= docker.io
+image_name = ${registry_url}/platform9/cluster-autoscaler/addon-resizer
+DOCKERFILE?=$(CURDIR)/Dockerfile
+UPSTREAM_VERSION?=$(shell git describe --tags HEAD | sed 's/-.*//' )
+image_tag = $(UPSTREAM_VERSION)-pmk-$(TEAMCITY_BUILD_ID)
+PF9_TAG=$(image_name):${image_tag}
+DOCKERARGS=
+ifdef HTTP_PROXY
+	DOCKERARGS += --build-arg http_proxy=$(HTTP_PROXY)
+endif
+ifdef HTTPS_PROXY
+	DOCKERARGS += --build-arg https_proxy=$(HTTPS_PROXY)
+endif
+
 GOARM=7
-GOLANG_VERSION = 1.16
+GOLANG_VERSION = 1.19
 REGISTRY = gcr.io/k8s-staging-autoscaling
 IMGNAME = addon-resizer
 IMAGE = $(REGISTRY)/$(IMGNAME)
 MULTI_ARCH_IMG = $(IMAGE)-$(ARCH)
-TAG = 1.8.16
+TAG = 1.8.16-pmk
```

Comment:
Can we add a suffix for Teamcity build ID here? Refer @jayanth-tjvrr 's changes in coredns repo

Reply:
Shaunak, let's update this too - to be consistent. Otherwise, rollback these changes. I don't think we should update this if we're not using it.